extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import '../amp-ad-exit';
+import {AmpAdExit} from '../amp-ad-exit';
```

Comment:
I don't see test coverage for the new code. 
The ideal test would be having an integration test that iframe sending message to <amp-ad-exit>. Maybe we can address it in separate PR.
But let's add unit tests to the methods you add in this PR.

Reply:
I added some test coverage in the other PR: https://github.com/ampproject/amphtml/pull/11462/files#diff-de8e41a56fd6ed0314e70ad7decdf9a0R548
But I agree, it would be good to have even more.